examples/seat-adjuster/Dockerfile

```diff
@@ -20,6 +20,7 @@ FROM --platform=$TARGETPLATFORM python:3.10-slim-bullseye@sha256:1ee6094f44c6778
 RUN apt-get update && apt-get install -y binutils
 
 COPY ./app /app
+COPY ./gen/vehicle_model /vehicle_model
```

Comment:
We also had the request in one hackathon to be able to build the dockercontainer without the devcontainer but the model was missing then.

Could we check here if the model is there and if not generate it on the fly?

Reply:
I guess, that's also what Mike Haller is currently missing.

Reply:
I think, it would be good if the Dockerfile would work "on their own": Means, we should enable to call `docker build <folder/uri of docker file>` and everything is fine. Ideally, a dockerfile is independent on some prerequisite steps to be executed before.

Reply:
Ticket created + plus keeping GH issue #84 open for final fix.